PICS.xcodeproj/project.pbxproj

```diff
@@ -951,7 +963,7 @@
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = 637867499T;
+				DEVELOPMENT_TEAM = "";
```

Comment:
I am not sure whether we to change this to empty here (and two changes after) Maybe we should be fine if the tests are passing? Otherwise, we may want to try revert the three changes here (and two after)